app/Lines/TreeLine.tsx

```diff
@@ -171,12 +177,28 @@ function NodeOther({ value }: { value: JsonValue }) {
     return <StringValue value={value} />;
   }
 
+  if (typeof value === "symbol") {
+    return (
+      <JSContainer>Symbol.for({JSON.stringify(value.description)})</JSContainer>
+    );
+  }
+
   if (
-    value !== null &&
     typeof value === "object" &&
-    Array.isArray(value) === false &&
+    !Array.isArray(value) &&
     !(value instanceof Array)
   ) {
+    if (isLazyReference(value)) {
+      return <ComponentTreeReference reference={value} />;
```

Comment:
These might need some UI adjustments -- in other places we're using `ComponentTreeReference`as a annotation for other content, and here it's used as the sole representation of a node which is kinda confusing

Reply:
Can you add a screenshot here?

Reply:
Going to set up a Storybook soon I think.